### PR TITLE
SNP for calico v2.4 or older

### PR DESCRIPTION
**Pull request by gunjan5**
## Description

Add SNP again for Calico v2.4 or older systems.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: hack/remove-calico-policy/override-policy.md [hunk 1]
```diff
@@ -19,17 +19,33 @@ ingress and egress traffic, overriding any other Network Policy that has been co
...
-Create a `GlobalNetworkPolicy` resource by running the following command:
+Create a `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource by running the following command:
+
+For Calico `v2.5.x` or higher:
 
 ```
 kubectl create -f=https://raw.githubusercontent.com/projectcalico/calico/master/hack/remove-calico-policy/global-network-policy-override.yaml
 ```
 
+For Calico `v2.4.x` or older:
```
Comment (robbrockbank) [inline hack/remove-calico-policy/override-policy.md]: Should we just say "v2.3.x and v2.4.x" (same applies below)

File: hack/remove-calico-policy/override-policy.md [hunk 2]
```diff
@@ -19,17 +19,33 @@ ingress and egress traffic, overriding any other Network Policy that has been co
 
 #### Override Calico policy to allow all traffic
 
-Create a `GlobalNetworkPolicy` resource by running the following command:
+Create a `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource by running the following command:
```
Comment (robbrockbank) [inline hack/remove-calico-policy/override-policy.md]: Maybe just reword this to something like:  "Create the policy-override resource by running..."

File: hack/remove-calico-policy/override-policy.md [hunk 3]
```diff
@@ -19,17 +19,33 @@ ingress and egress traffic, overriding any other Network Policy that has been co
...
+For Calico `v2.4.x` or older:
+
+```
+kubectl create -n=kube-system -f=https://raw.githubusercontent.com/projectcalico/calico/master/hack/remove-calico-policy/system-network-policy-override.yaml
+```
+
 #### Revert override to enable Calico policy
 
-To revert the override of Calico policy, delete the `GlobalNetworkPolicy` resource 
+To revert the override of Calico policy, delete the `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource
```
Comment (robbrockbank) [inline hack/remove-calico-policy/override-policy.md]: again, perhaps easier here just to say - to remove the policy-override resource..

File: hack/remove-calico-policy/system-network-policy-override.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+apiVersion: "alpha.projectcalico.org/v1"
+kind: SystemNetworkPolicy
+metadata:
+  name: allow-all-calico-policy
```
Comment (robbrockbank) [inline hack/remove-calico-policy/system-network-policy-override.yaml]: Is this correct?  Shouldn't it be in the kube-system namespace?

Comment (robbrockbank) [inline hack/remove-calico-policy/system-network-policy-override.yaml]: Oh you've set the namespace in the kubectl command... okey doke.

Comment (gunjan5) [inline hack/remove-calico-policy/system-network-policy-override.yaml]: I just copied the original manifest, but I think it's better to make namespace part of the manifest. will do that
